nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
```

Comment:
You missed the point. We're not using the 'XREF' value, this uses a corrupted version of it. We shouldn't be doing that. Instead we should store that data somewhere else, like a new table or column.